.eslintignore

```diff
@@ -0,0 +1 @@
+test/async-function-tests.js
```

Comment:
This was needed because our version of `eslint` cannot parse `async function() {}` properly.